### ISSUE TITLE
struct2depth is the kitti dataset generator including test files in training set?

### ISSUE THREAD

**Issue by aj96** [tensorflow/models#6978]
### System information
- **What is the top-level directory of the model you are using**: struct2depth
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**: No
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**: Ubuntu 18.04
- **TensorFlow installed from (source or binary)**: pip install tensorflow-gpu
- **TensorFlow version (use command below)**: 1.13.1
- **Bazel version (if compiling from source)**: N/A
- **CUDA/cuDNN version**: 10.0 / 7.4.1
- **GPU model and memory**: Tesla P-100 16 Gb
- **Exact command to reproduce**: N/A

It seems that the gen_data_kitti.py  is not excluding the test set when creating the training and validation sets. Can anyone else confirm this?

**Comment by amrbenattia** [comment]
Hi @aj96 
I am trying to create kitti dataset, could you help how did you run gen_data_kitti.py
thanks in advance

**Comment by dhiren-hamal** [comment]
Dataset generator is not working in mine too, I tried to run `python gen_data_kitti.py` but it looks like only folders got generated, any Idea how to generate the training files?

**Comment by kumariko** [comment]
@aj96  We see you are using an older version of tensorflow(1.x) which is officially considered as end of life. We recommend that you upgrade to 2.7 which is latest version and let us know if the issue still persists in newer versions. Thanks!
